terraform/modules/marklogic/backup_buckets.tf

```diff
@@ -48,10 +48,8 @@ resource "aws_s3_bucket_lifecycle_configuration" "weekly_backup_bucket" {
       days_after_initiation = 14
     }
 
-    # There shouldn't be many noncurrent objects since the expiration rule will delete them directly,
```

Comment:
Turns out this just isn't true, it actually adds a delete marker. Don't know where I got that from!

See https://docs.aws.amazon.com/AmazonS3/latest/userguide/intro-lifecycle-rules.html#w2010aac25c23c30c23c25